suitcase/tiff_series/tests/tests.py

```diff
@@ -215,3 +215,30 @@ def return_4d_data():
 
     filenames = os.listdir(path=tmp_path)
     assert len(filenames) == 0
+
+
+def test_2d_data_lies(RE, tmp_path):
+    """
+    Expect one file written for each detector trigger.
+    """
+    def return_2d_data():
+        return np.zeros((3, 3))
+
+    class LyingDirectImage(DirectImage):
+        def describe(self):
+            ret = super().describe()
+            ret['img']['shape'] = (1, ) + ret['img']['shape']
+            return ret
+
+    det = DirectImage(name="2d_image", func=return_2d_data)
```

Comment:
Is it supposed to instantiate from `LyingDirectImage`?
```suggestion
    det = LyingDirectImage(name="2d_image", func=return_2d_data)
```